engine/apps/alerts/tasks/check_escalation_finished.py

```diff
@@ -29,26 +29,26 @@ def send_alert_group_escalation_auditor_task_heartbeat() -> None:
 
 
 def audit_alert_group_escalation(alert_group: "AlertGroup") -> None:
-    escalation_snapshot = alert_group.escalation_snapshot
+    raw_escalation_snapshot: dict = alert_group.raw_escalation_snapshot
```

Comment:
To be sure, the difference here is we use the raw representation to get the information instead of going through the deserialized version?

Reply:
that's correct, we get information directly from alert group json field without serializing and getting additional objects from database, since we don't need it here.